microsoft/nni
# [Compression] support block sparse in level pruner

### Description ###
Split `LevelPruner` to `LevelPruner` and `BalancedPruner`, support `align_n` in `BalancedPruner`, add `BlockPruner`.

### Checklist ###
- [x] doc (partial)

### How to test ###

set (`align_n` in `BalancedPruner`) & (`block_sparse_size` in `BlockPruner`) and check the generated masks to see if it is block sparse.
